modelskill/comparison/_comparison.py

```diff
@@ -458,7 +485,8 @@ def __repr__(self):
             f"Observation: {self.name}, n_points={self.n_points}",
         ]
         for model in self.mod_names:
-            out.append(f" Model: {model}, rmse={self.sel(model=model).score():.3f}")
+            out.append(f" Model: {model}, rmse={self.score()[model]:.3f}")
```

Comment:
there is potentially a performance hit here, and maybe we should only do this if n_points < 10^5 or similar? (we don't want repr to be slow)

Reply:
Or not at all?